Website/src/items/content/EdgeWebsite/home.yml

```diff
@@ -1,184 +1,184 @@
 ﻿---
-ID: "2d1d7aca-9597-5ca6-85b2-9b04907e4298"
-Parent: "6f510288-14c5-5dc8-a8e9-e4205fd03dd3"
-Template: "6db09cef-64a8-52a3-a980-38408109c4d7"
+ID: '2d1d7aca-9597-5ca6-85b2-9b04907e4298'
```

Comment:
Please do not use a YAML file prettier/linter on serialized items. We try to keep them formatted the way Sitecore Content Serializer (SCS) is saving them. Please revert the YML files linting changes in this PR:
- Quote style
- Indentation
- Explicit closing of self-closing XML elements

Thanks!